inc/dbconnection.class.php

```diff
@@ -470,4 +470,4 @@ static function changeCronTaskStatus($enable=true) {
    }
 
 }
-?>
\ No newline at end of file
+?>
```

Comment:
please, don't change, this, quash your commit, and everything will be fine for me.